omero/developers/Web/Deployment.txt

```diff
@@ -80,7 +81,6 @@ listening on 127.0.0.1:4080 that will autoreload on source change:
 ::
 
     $ nginx -c $PWD/nginx-development.conf
-    $ bin/omero config set omero.web.application_server 'wsgi-tcp'
     $ bin/omero config set omero.web.application_server.max_requests 1
     $ bin/omero web start  --wsgi-args ' --reload'
```

Comment:
Since PR #4352 is on hold and it's separate from the bug that this PR is addressing, let's fix it in a different PR.